docker-secrets.yml

```diff
@@ -131,12 +131,3 @@ keys:
       V7ZuRp//4vIzEONqqyLUFt1JZKuCwWQB2sRjKWpqRB07C0o243XefD5f7844nmXq
       H0xKMFRYjDqhFdTXFWVPnk/tu/MxoSTdK0FfsgUnjs+TGvN5TnBW
       -----END RSA PRIVATE KEY-----
-secrets:
```

Comment:
You should also remove `EDCRRM` and `EQ_USER_AUTHENTICATION_SR_PRIVATE_KEY` from this file